src/main/modules/i18n/index.ts

```diff
@@ -27,7 +27,7 @@ export class I18Next {
           caches: ['cookie']
         },
         interpolation: {
-          escapeValue: false
+          escapeValue: false // Escaping is already handed by Nunjucks
```

Comment:
handled

Reply:
@rlewan   Add a comment to explain why the change is needed  ea122d9
 @rlewan   Fix typo   ea3e9e5
😎